Makefile

```diff
@@ -24,15 +25,19 @@ bootstrap:
 
 .PHONY: test-integration
 test-integration:
-	go test ./tests/... ${TEST_OPTS}
+	DEFAULT_EVENTUALLY_TIMEOUT="30s" ginkgo $TEST_OPTS tests/
```

Comment:
The default in master for sequential tests is `10s` and it has proven to be _relatively_ consistent with that value.  Can we keep it at 10?